contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1710,28 +1710,28 @@ The following guidelines apply to all "Additional resources" sections:
 
 * You must include the `[role="_additional-resources"]` attribute declaration before the section heading.
 * You must not include paragraphs in the section. Use an unordered list.
-* The links and xrefs in the unordered list must contain human-readable text between the square brackets.
-* Each item in the unordered list must contain a minimum of text besides the link or xref.
+* The links and xrefs in the unordered list must contain minimal-length, human-readable text between the square brackets. This text is often the title of the linked page.
+* You must not include text outside of the square brackets.
 
 Additionally, in an assembly, use `==` formatting for the section heading (`== Additional resources`). Use of this heading syntax at the assembly level indicates that the sections relate to the whole assembly. For example:
 
 ----
 [role="_additional-resources"]
 [id="additional-resources_configuring-alert-notifications"]
 == Additional resources
-* link:some-url.com[Human readable label]
-* xref:some_xref[Human readable label]
-* xref:some_other_xref[Human readable label]
+* link:example.com[IANA example domain for documentation]
+* xref:installation/installing-the-product.adoc#installing-the-product[Installing the product]
```

Comment:
IMO, all of the xrefs should be offset one level. Otherwise, this LGTM.
```suggestion
* xref:../installation/installing-the-product.adoc#installing-the-product[Installing the product]
```

Reply:
oh yeah true, they are always gonna look like that in our work :) Will do!